README.md

```diff
@@ -1,31 +1,86 @@
-# Guidelines
-  **Installation**
-  1. npm install
-  2. npm run build-prod
-  3. npm run build-dev
-  4. npm start
+# Front End Capstone
+> Table of Contents
 
-## Check out
-  **Upgrading from Enzyme 2.x or React < 16**
-    - To get started with enzyme, you can simply install it via npm. You will need to install enzyme along with an Adapter corresponding to the version of react
+- [Introduction](#Introduction)
+- [Technologies](#Technologies)
+- [Components guide](#Components)
+- [Installation Guidelines](#Installation-Guidelines)
+- [Resource](#Resource)
 
-**We are going to use "react": "^17.0.1"**
+## Introduction
+> Developed by Aharon Kolatch, Cody Jue, Will Chang and Claire Sujin Kim
+```
+We developed this project as a group over the course of 9 days during the Hack Reactor program.
+We were all worked on our own individual widget for the site.
 
-    import Enzyme from 'enzyme';
-    import Adapter from 'enzyme-adapter-react-16';
+built up based on given a business document outlined a strict set of requirements that guided
+our implementation, functionality, and styling.
+```
+- **Aharon** - Product Overview
+- **Claire** - Related Items and Comparison
+- **Cody** - Questions and Answers
+- **Will** - Ratings and Reviews
 
-    Enzyme.configure({ adapter: new Adapter() });
+![catwalk](img/fec.gif)
 
+## Technologies
+For this project, we used the following languages and frameworks in our implementation
+```bash
+- Javascript
+- React
+- Redux
+```
+> Deployment
+We deployed our site on AWS, which can be viewed **here**
 
-    *** more info: https://www.npmjs.com/package/enzyme
+## Components
+business document can be found [here](https://docs.google.com/document/d/1KAqduzY8ae3DYrSoCL1i23qHe95zJRYFulqMk-sGLWY/edit#).
```

Comment:
I think we probably should leave the business doc out. It's Hack Reactor's, so they probably prefer it to be kept secret.

Reply:
Sounds right!!